src/components/ControlPanel.jsx

```diff
@@ -25,28 +44,65 @@ class ControlPanel extends React.Component {
       <FieldGuide guide={this.props.guide} icons={this.props.icons} />, 'Field Guide'));
   }
 
-  render() {
-    const fieldGuideText = this.props.dialog ? 'Hide Field Guide' : 'Show Field Guide';
+  handleResize() {
+    if (window.innerHeight < 625 && this.state.showInfo) {
```

Comment:
A part of me thinks that this hardcoded size-related value should be extracted out to a config file... a... `secondary-config-for-styles-i-guess.js` file, sorta. But another part of me just shrugs and thinks, this value is only used in this one component.

I'm going to OK this for now, but I have a feeling this may bite us in the far future.

Reply:
Yeah, it's hard to say. I can put it in a config file or at least make it a constant in the component if you think that'd help.